postgre-client/src/asset_filter_client.rs

```diff
@@ -123,7 +123,7 @@ impl PgClient {
                     ast_slot_created,
                     ast_slot_updated
                     FROM assets_v3
-                    JOIN fungible_tokens ON ast_pubkey = fungible_tokens.fbt_asset
+                    JOIN fungible_tokens ON ast_pubkey = fungible_tokens.fbt_asset AND ast_owner_type = 'token'
```

Comment:
Can there be an asset with **ast_owner_type** **single** or **unknown**? (because as far as I understand we apply this change to all possible requests)

Reply:
This PR https://github.com/metaplex-foundation/aura/pull/376 adds fix not to have `unknown` in `ast_owner_type` anymore.
And we join only rows with `ast_owner_type = 'token'` because for NFTs(rows with `ast_owner_type = 'single'`) there is no necessary information in `fungible_tokens` table.

Reply:
If we join rows with `ast_owner_type='single'` we start to return NFTs with zero balance, which was identified as unwanted behaviour.

Reply:
How are the indexes being used for this join?